repos/fdbt-site/src/pages/products/otherProducts.tsx

```diff
@@ -77,7 +77,11 @@ const otherProductsTable = (
 ): React.ReactElement => {
     return (
         <>
-            <table className="govuk-table" data-card-count={otherProducts.length}>
+            <table
+                className="govuk-table"
+                data-card-count={otherProducts.length}
+                data-period-card-count={otherProducts.filter((product) => product.type === 'period').length}
```

Comment:
Can we not figure this out via cypress instead of adding it here?

Reply:
we need to iterate the table and check for the product type, but using this it will give me the count without loop

Reply:
Yeah - I think we should stop adding code to the site to enable 'easier' UI tests. We should be using cypress to figure out both the number of products on this page, and the number of times it says 'period' on the page.